packages/repository/cookies.ts

```diff
@@ -0,0 +1,22 @@
+import Cookies from 'universal-cookie';
+
+type cookie = {
+  key: string;
+  value: string;
+};
+
+const cookies = new Cookies();
+
+function storeCookie({ key, value }: cookie) {
+  cookies.set(key, value, { path: '/' });
+}
```

Comment:
```suggestion
}

```
Please add an empty blank line to separate functions